src/main/java/com/example/controllers/userControllers/NotesController.java

```diff
@@ -0,0 +1,66 @@
+package com.example.controllers.userControllers;
+
+import com.example.entity.Notes;
+import com.example.dto.RestResponseDto;
+import com.example.services.NotesService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.boot.json.JsonParseException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+import java.util.List;
+import javax.annotation.Resource;
+import javax.persistence.Lob;
+
```

Comment:
JD

Reply:
@Vlada0612 Following your advice the java dock was added.